pkg/notifier/notifier.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package notifier;
+option go_package = "./;notifier";
+
+message Empty {}
+
+message ConfigsRequest {
+    map<string,string> configs = 1;
+}
+
+service Notifier {
+    rpc LoadConfigs(ConfigsRequest) returns (Empty) {};
```

Comment:
mmm generally we create messages for request or response, no matter if they are empty, 
but when we want to add fields in a future, it keep working without back compatibility issues.
So I suggest to add messages for both requests and response